fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/rdf/RdfStream.java

```diff
@@ -13,12 +13,23 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+package org.fcrepo.kernel.api.rdf;
```

Comment:
Erm, mebbe. See my note [here](https://github.com/fcrepo4/fcrepo4/pull/962/files#diff-d21cc8996ca7ecac4305348e5792e256R55). I think `RdfContext` and `RdfCollectors` are more like API, but not `DefaultRdfStream`. Sort of.